hack/bootstrap-kubestellar.sh

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: $0  --verbose
+
+# This script installs kcp and KubeStellar binaries to a folder of choice
+#
+# Arguments:
+# [--kcp-version release] set a specific kcp release version, default: latest
+# [--kubestellar-version release] set a specific KubeStellar release version, default: latest
+# [--os linux|darwin] set a specific OS type, default: autodetect
+# [--arch amd64|arm64] set a specific architecture type, default: autodetect
+# [--folder name] sets the installation folder, default: $PWD/kcp-edge
+# [--bind address] bind kcp to a specific ip address
+# [--imw name] create a Inventory Management Workspace (IMW)
+# [--wmw name] create a Workload Management Workspace (WMW)
```

Comment:
I think that naming these flags `--ensure-imw` and `--ensure-wmw` would make for command lines that read better.  Adding the verb says more clearly that the identified workspaces are to be created if not existing, rather than having some other relationship to what is going on here (e.g., expected to be pre-existing).

Reply:
ok, done